### ISSUE TITLE
Bring up SDXL

### ISSUE THREAD

**Issue by tengyifei** [AI-Hypercomputer/torchprime#184]
This is a larger/more advanced image model compared to Stable Diffusion 2.1

**Comment by tengyifei** [comment]
It could be nice to keep a reference impl that's E2E tested and don't bit rot over time. This seems like a constant customer ask.

**Comment by bhavya01** [comment]
I will break down the work into following steps:

* ClipTextEncoder implementation + compare output with huggingface
* VAE implementation + compare output with huggingface
* Unet implementation + compare output with huggingface
* Training script + compare loss with huggingface
